.github/workflows/InitDev,Unit tests,Integration tests,Package.yml

```
@@ -48,14 +49,38 @@ jobs:
       working-directory: ./Ed-Fi-ODS-Implementation/
       shell: pwsh
       run: |
-            $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
-            Write-Host "Pattern Name is " $patternName -fore GREEN
-            $is_pull_request_branch = 'False'
-            $is_pull_request_branch = git ls-remote --heads origin ${{ env.CURRENT_BRANCH }} | Select-String -Pattern $patternName -SimpleMatch -Quiet
-            if ($is_pull_request_branch -eq $true) {
-              git fetch origin ${{ env.CURRENT_BRANCH }}
-              git checkout ${{ env.CURRENT_BRANCH }}
-            }
+           $odsBranch = '${{ env.REPOSITORY_DISPATCH_BRANCH }}'
+           Write-Host "OdsBranch: $odsBranch"
```

https://github.com/PowerShell/PSScriptAnalyzer/blob/master/docs/Rules/AvoidUsingWriteHost.md

```suggestion
           Write-Output "OdsBranch: $odsBranch"
```